src/components/common/view/TopBar.js

```diff
@@ -13,21 +13,21 @@ import BigGoodDollar from './BigGoodDollar'
  * @param {React.Node} props.children
  * @returns {React.Node}
  */
-const TopBar = ({ hideBalance, push, children }) => {
+const TopBar = ({ hideBalance, push, children, hideProfile = true }) => {
```

Comment:
i've read about latest updates in .defaultProps
for now the recommended way to set them is as you're doing
**but if default value is an object-like** (object literal, array etc) you should **define it as constant above** and specify the constant defined as default value

e.g.

```
const defaultOptions = {};
​
const MyComponent = ({ enabled = false, options = defaultOptions }) => {
}
```

Reply:
10x